packages/runtime/container-runtime/src/summary/runningSummarizer.ts

```diff
@@ -243,12 +243,20 @@ export class RunningSummarizer extends TypedEventEmitter<ISummarizerEvents> impl
 			}
 		});
 
+		const immediatelyRefreshLatestSummaryAck =
+			this.mc.config.getBoolean("Fluid.Summarizer.immediatelyRefreshLatestSummaryAck") ??
+			false;
 		this.generator = new SummaryGenerator(
 			this.pendingAckTimer,
 			this.heuristicData,
 			this.submitSummaryCallback,
-			() => {
+			async (options: IRefreshSummaryAckOptions) => {
 				this.totalSuccessfulAttempts++;
+				if (immediatelyRefreshLatestSummaryAck) {
```

Comment:
Given the `successfulSummaryCallback` is no longer a short one line code-block; consider defining this Callback above line 249 and just passing in here, similiar to how we do `submitSummaryCallback`

Reply:
I thought of doing that as well. Sure I can do that.

Reply:
I reduced the refresh call to one line.